fhd_utils/skyh5_to_fhd.pro

```diff
@@ -0,0 +1,42 @@
+; Script to read .skyh5 files output by pyradiosky in an FHD-compatible way
+; Written by Dara Storer, 5/24/21
+
+pro skyh5_to_fhd, filename, filename_sav
```

Comment:
Most of the code here duplicates (or is duplicated by) `load_skyh5_source_catalog`. Please combine the two into one function, or if that is not possible then factor out as much of the common code as possible into a helper function.